bench/fixture.ts

```diff
@@ -323,4 +325,60 @@ export class BenchFixture {
 
     return await transaction.wait();
   }
+
+  public async settleOne(): Promise<ContractReceipt> {
+    const {
+      deployment: { settlement },
+      domainSeparator,
+      solver,
+      traders: [trader],
+      uniswapPair,
+      uniswapTokens: [sellToken, buyToken],
+    } = this;
+
+    const sellAmount = ethers.utils.parseEther("1.0");
+    const buyAmount = ethers.utils.parseEther("0.9");
+    const feeAmount = sellAmount.div(1000);
+
+    const order = {
+      sellToken: sellToken.address,
+      buyToken: buyToken.address,
+      sellAmount,
+      buyAmount,
+      validTo: 0xffffffff,
+      appData: this.nonce++,
```

Comment:
I thought appData was meant for storing info like which frontend placed the order. Is this just an arbitrary choice for appData or is this somehow relevant?

Reply:
Its just used here to so orders are guaranteed to be different (since it uses the same parameters for all benchmarks and so the hash/signature would be the same).